os_builders/roles/vm_baseline/files/update_keys.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if ls /home/cloud
+then
+    KEYSPATH=/home/cloud
+else
+    KEYSPATH=/root
+fi
+
+if lsattr $KEYSPATH/.ssh/authorized_keys | grep "\-i\-"
+then
+    echo "file $KEYSPATH/.ssh/authorized_keys is immutable, cannot write admin key list to it. Generated from \`update_keys.sh\`." #| mail -s "Error during VM authorized_keys update" cloud-support@stfc.ac.uk
```

Comment:
This outer if / else can be removed if we simply
`exit 1` after echoing

Reply:
Possibly but I dont think that's as readable